docs/business.yaml

```diff
@@ -2712,41 +3097,122 @@ components:
       description: 'Add or remove a director, or update information regarding a current director.'
       properties:
         changeOfDirectors:
-          type: array
-          description: 'Add or remove a director, or update information regarding a current director.'
-          items:
-            $ref: '#/components/schemas/party'
+          type: object
+          description: 'This section contains all the changes you want to make to your company''s directors.'
+          required:
+            - directors
+          properties:
+            directors:
+              type: array
+              items:
+                allOf:
```

Comment:
Maybe consider having `director` and `actions` as 2 separate schemas and reference them in the CoD? Since, there is a chance that other places need to use them in the future.
![image](https://github.com/user-attachments/assets/5b408109-6605-4744-a1ba-944d72b534f6)

Reply:
Yes, I noticed that the `directors` schema is also used in the annual report. Thank you for pointing this out. I think creating a separate directors schema based on the structure in the [business schema](https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/directors.json) would be a good idea.

Regarding `actions`, I am not sure if we should create a separate schema since right now this definition is only used in the change of directors.

@chenhongjing  @argush3  Hope we can get your insights or suggestions on this matter when you have a moment.

Reply:
Yeah, I think you are right, the `actions` is only used in CoD, since it's from the definitions, not from a separate schema. 
For the `directors`, probably creating a separate one and update the places using $ref is a good idea?

Reply:
I think it's good to create a separate schema for `director` while keeping `actions` within COD.